crates/rome_js_formatter/tests/specs/js/module/export/named_from_clause.js

```diff
@@ -6,4 +6,7 @@ export {a,
 export {
     lorem_lorem_lorem_lorem_lorem_lorem_lorem_lorem_lorem,
     lorem_lorem_lorem_lorem_lorem_ as ipsum_ipsum_ipsum_ipsum_ipsum_ipsum_
-} from "fancy" assert { type: "json", "type2": "json", type23: "json", "type24": "json"}
\ No newline at end of file
+} from "fancy" assert { type: "json", "type2": "json", type23: "json", "type24": "json"}
+
+
+export { loooooooooooooooooooooooooooooooooooooooooooooooooong } from "loooooooooooooooooooooooooooooooooooooooooooooong"
```

Comment:
Let's add some test case with  a trailing `,` and some comments
```suggestion
export { loooooooooooooooooooooooooooooooooooooooooooooooooong } from "loooooooooooooooooooooooooooooooooooooooooooooong"

export { loooooooooooooooooooooooooooooooooooooooooooooooooong, } from "loooooooooooooooooooooooooooooooooooooooooooooong"

export { 
// comment 
loooooooooooooooooooooooooooooooooooooooooooooooooong } from "loooooooooooooooooooooooooooooooooooooooooooooong"
```